lsp-ui-sideline.el

```diff
@@ -234,7 +234,7 @@ function signature)."
 If the string has a language, we fontify it with the function provided
 by `lsp-mode'.
 MARKED-STRING is the string returned by `lsp-ui-sideline--extract-info'."
-  (when (and marked-string (lsp-marked-string? marked-string))
+  (when (and marked-string (or (lsp-marked-string? marked-string) (lsp-markup-content? marked-string)))
```

Comment:
Could you update the PR description explaining the problem with the Metals language server (that should be part of the commit description as well)? That'd help us understand some changes in the future if we need to revisit why something was done.

Reply:
Updated the commit message.

Reply:
Looks like github completely missed my commit so pushed once again.